Makefile.RAHasher

```diff
@@ -47,6 +50,10 @@ $(OUTDIR)/RAHasher$(EXE): $(OBJS)
 src/Git.cpp: etc/Git.cpp.template FORCE
 	cat $< | sed s/GITFULLHASH/`git rev-parse HEAD | tr -d "\n"`/g | sed s/GITMINIHASH/`git rev-parse HEAD | tr -d "\n" | cut -c 1-7`/g | sed s/GITRELEASE/`git describe --tags | sed s/\-.*//g | tr -d "\n"`/g > $@
 
+zip:
```

Comment:
You should make this explicitly dependent on `$(OUTDIR)/RAHasher$(EXE)`. Then if you did a `make zip` it would make sure the exe is up to date before zipping it up.